actuators/machine.go

```diff
@@ -197,41 +191,41 @@ func (m *Machine) Exists(ctx context.Context, cluster *clusterv1.Cluster, machin
 		fmt.Sprintf("label=%s=%s", constants.ClusterLabelKey, cluster.Name),
 		fmt.Sprintf("name=^%s$", machine.GetName()),
 	}
-	fmt.Printf("using labels: %v\n", labels)
+	m.Log.Info("using labels", "labels", labels)
 	nodeList, err := nodes.List(labels...)
 	if err != nil {
 		return false, err
 	}
-	fmt.Printf("found nodes: %v\n", nodeList)
+	m.Log.Info("found nodes", "nodes", nodeList)
 	return len(nodeList) >= 1, nil
 }
 
 // patches the object and saves the status.
 func (m *Machine) save(old, new *clusterv1.Machine, noderef *apicorev1.ObjectReference) error {
```

Comment:
Instead of `new` as a variable name, how about `originalMachine` and `machineToSave` or something along those lines?

Reply:
yep, much better, but out of scope for this PR, maybe open an issue to track potentially confusing naming

Reply:
#31